app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -1,5 +1,34 @@
 /* list
 ---------------------------------------------------------------------------- */
+.list-group {
+  &.list-group-settings_list {
+    margin-bottom: 0;
+
+    .list-group-item {
+      border-radius: 0;
+      border: 0;
+      border-left: 5px solid transparent;
+      color: $color-forever_blue;
+      background-color: #f9f8f7;
+
+      &:hover {
+        color: $color-gray-600;
+        background-color: #e7e9e6;
+      }
+      &.active {
+        background-color: #e7e9e6;
```

Comment:
I didn't see variables for these colors (`#e7e9e6` and `#f9f8f7`) because they're more closely tied to the current design of the web app, rather than the style guide. Let me know if it would be better to remove them and just add them in the web app.